.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: om6dV5LyaA4SdwT67Jn9W3w1fZCF10uUe
```

Comment:
**Nitpick**
Am I not sure you want to do this? This can be very harmful.
Besides, with Travis(And public repository), all you need to do is add your repo to the platform and each build will trigger a report to coveralls.